README.md

```diff
@@ -58,9 +58,6 @@ Databases on SQL Server, Azure SQL Database, or Azure SQL Managed Instance which
   - PowerShell: The workaround is to use the `$TriggerMetadata[$keyName]` to retrieve the query property - an example can be found [here](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-powershell/AddProductParams/run.ps1). Issue is tracked [here](https://github.com/Azure/azure-functions-powershell-worker/issues/895).
   - Python: The workaround is to use `parse_qs` - an example can be found [here](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-python/AddProductParams/__init__.py). Issue is tracked [here](https://github.com/Azure/azure-functions-python-worker/issues/894).
 
-### Input Bindings
-- For Java Functions using Output bindings against a table with columns of data types 'DATETIME', 'DATETIME2', or 'SMALLDATETIME', use java.util.Date type to ensure the datetime is formatted correctly.
```

Comment:
Meant to remove this when adding known issues to Java guide https://github.com/Azure/azure-functions-sql-extension/blob/main/docs/SetupGuide_Java.md#known-issues